packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -389,7 +391,7 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         // library for details.
         bytes memory proxyBytecode = hex"67363d3d37363d34f03d5260086018f3";
 
-        address proxy = computeCreate2Address(_salt, keccak256(proxyBytecode), _deployer);
+        address proxy = vm.computeCreate2Address(_salt, keccak256(proxyBytecode), _deployer);
```

Comment:
I had to prepend `vm` because Forge [recently deprecated the standard `computeCreate2Address` function](https://github.com/foundry-rs/forge-std/blob/master/src/StdUtils.sol#L106-L114)